samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -414,6 +414,29 @@ mfxStatus CEncodingPipeline::InitMfxEncParams(sInputParams *pInParams)
 
     if (m_mfxEncParams.mfx.RateControlMethod == MFX_RATECONTROL_CQP)
     {
+        bool enableRepackCtrl = pInParams->repackctrlFile != NULL && pInParams->CodecId == MFX_CODEC_AVC;
+
+        if (m_pRepackCtrl_in == NULL && enableRepackCtrl)
+        {
+            MSDK_FOPEN(m_pRepackCtrl_in, pInParams->repackctrlFile, MSDK_CHAR("rb"));
+            if (m_pRepackCtrl_in == NULL)
+            {
+                msdk_printf(MSDK_STRING("ERROR: Can't open file %s\n"), pInParams->repackctrlFile);
+                return MFX_ERR_UNSUPPORTED;
+            }
+            msdk_printf(MSDK_STRING("Using Frame Size control input file: %s\n"), pInParams->repackctrlFile);
+
+            SAFE_FREAD(&(m_mfxEncParams.mfx.MaxIFrameSize), sizeof(mfxU32), 1, m_pRepackCtrl_in, MFX_ERR_MORE_DATA);
+            SAFE_FREAD(&(m_mfxEncParams.mfx.MaxPFrameSize), sizeof(mfxU32), 1, m_pRepackCtrl_in, MFX_ERR_MORE_DATA);
```

Comment:
Much better using object name in sizeof than its implicit type.

Reply:
Will replace the type with object name.